controllers/admin_domains.php

```diff
@@ -747,7 +763,20 @@ public function tlds()
             $params = array_merge($vars, $params);
 
             if (!empty($vars['module'])) {
-                $params['module_id'] = (int) $vars['module'];
+                $params['module_id'] = (int)$vars['module'];
+            } else {
```

Comment:
The point of removing this kind of functionality from the model was to simplify the code and remove special cases.  We don't need this special case either.

If $vars['module'] is not set then the model will simply return an error, which is fine.